demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -291,6 +291,120 @@ def _collect_playbooks_dependencies(pack_playbooks, id_set):
 
         return dependencies_packs
 
+    @staticmethod
+    def _collect_layouts_dependencies(pack_layouts, id_set):
+        """
+        Collects layouts pack dependencies.
+
+        Args:
+            pack_layouts (list): collection of pack playbooks data.
+            id_set (dict): id set json.
+
+        Returns:
+            set: dependencies data that includes pack id and whether is mandatory or not.
+
+        """
+        dependencies_packs = set()
+
+        for layout in pack_layouts:
+            layout_data = next(iter(layout.values()))
+
+            related_incident_and_indicator_types = layout_data.get('incident_and_indicator_types', [])
+            packs_found_from_incident_indicator_types = PackDependencies._search_packs_by_items_names(
+                related_incident_and_indicator_types, id_set['IncidentTypes'] + id_set['IndicatorTypes'])
+
+            if packs_found_from_incident_indicator_types:
+                pack_dependencies_data = PackDependencies. \
+                    _label_as_mandatory(packs_found_from_incident_indicator_types)
+                dependencies_packs.update(pack_dependencies_data)
+
+            related_incident_and_indicator_fields = layout_data.get('incident_and_indicator_fields', [])
+            packs_found_from_incident_indicator_fields = PackDependencies._search_packs_by_items_names(
+                related_incident_and_indicator_fields, id_set['IncidentFields'] + id_set['IndicatorFields'])
+
+            if packs_found_from_incident_indicator_fields:
+                pack_dependencies_data = PackDependencies. \
+                    _label_as_mandatory(packs_found_from_incident_indicator_fields)
+                dependencies_packs.update(pack_dependencies_data)
+
+        return dependencies_packs
+
+    @staticmethod
+    def _collect_incidents_fields_dependencies(pack_incidents_fields, id_set):
+        """
+        Collects in incidents fields dependencies.
+
+        Args:
+            pack_incidents_fields (list): collection of pack incidents fields data.
+            id_set (dict): id set json.
+
+        Returns:
+            set: dependencies data that includes pack id and whether is mandatory or not.
+
+        """
+        dependencies_packs = set()
+
+        for incident_field in pack_incidents_fields:
+            incident_field_data = next(iter(incident_field.values()))
+
+            related_incident_types = incident_field_data.get('incident_types', [])
```

Comment:
incident field is pointing at type and vice versa?

Reply:
No, incidents fields are depended on incident types and thats it.